### PR TITLE
Including new feature for TOF: Hit

### PR DESCRIPTION
**Pull request by njacazio**
- Added new feature for TOF: Hit  (F. Noferini)
- Added macro to show TOF geometry
- All addition are formatted after clang-format

### CODE REVIEW

File: Detectors/TOF/simulation/include/TOFSimulation/Detector.h [hunk 1]
```diff
@@ -62,13 +64,14 @@ class Detector : public o2::Base::Detector
...
   void Reset() final;
+  void EndOfEvent() final;
 
   void CreateMaterials();
   void ConstructGeometry() final;
 
   void SetTOFholes(Bool_t flag = kTRUE) { mTOFHoles = flag; }
  protected:
-  virtual void ConstructSuperModule(Int_t imodule) final;
+  HitType* AddHit(Float_t x, Float_t y, Float_t z, Float_t time, Float_t energy, Int_t trackId, Int_t detId);
```
Comment (sawenzel) [inline Detectors/TOF/simulation/include/TOFSimulation/Detector.h]: probably the return type is not needed ...

**Comment by sawenzel** [comment]
Thanks. This looks very good. Will merge as soon as we have 3 passing tests.

File: Detectors/TOF/prototyping/drawTOFgeometry.C [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#if (!defined(__CINT__) && !defined(__CLING__)) || defined(__MAKECINT__)
```
Comment (ktf) [inline Detectors/TOF/prototyping/drawTOFgeometry.C]: This is missing the copyright statement. Can you please add it? I think we are not yet checking for copyright in macros, but we probably should.

Comment (njacazio) [inline Detectors/TOF/prototyping/drawTOFgeometry.C]: Hi, I'm sorry for this. I started working from some already present macro i.e. Detectors/Passive/macro/PutFrameInTop.C which did not have the statement so I forgot. Adding it straight away! Should I open a new PR or include the commit in a future one?

**Pull request merged**